<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>單位轉換器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <widget class="QLineEdit" name="LineEditInput1">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>小賴字體 等寬 SC</family>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="ComboBoxInput2">
       <property name="font">
        <font>
         <family>小賴字體 等寬 SC</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="ComboBoxInput1">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>小賴字體 等寬 SC</family>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="LineEditInput2">
       <property name="font">
        <font>
         <family>小賴字體 等寬 SC</family>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="LabelOutput2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>小賴字體 等寬 SC</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="LabelOutput1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>小賴字體 等寬 SC</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(112, 112, 112);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QComboBox" name="ComboBoxSelect">
       <property name="font">
        <font>
         <family>小賴字體 等寬 SC</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonCalc">
       <property name="text">
        <string>計算</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
